examples/seismic/skew_self_adjoint/example_iso.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+from devito import info
```

Comment:
blank line between third-party imports and devito imports

Reply:
notebooks also?

Reply:
will defer nb to the coming big change for Model/Geom